src/vl.js

```diff
@@ -4,7 +4,10 @@ var globals = require('./globals'),
     util = require('./util'),
     consts = require('./consts');
 
-var vl = util.merge(consts, util);
+var vl = {};  // util.merge(consts, util);
```

Comment:
Will fix those. The problem is that if everything is in the same namespace it's really hard to find anything.